torax/_src/neoclassical/transport/base.py

```diff
@@ -64,22 +66,113 @@ def zeros(cls, geometry: geometry_lib.Geometry) -> 'NeoclassicalTransport':
+        runtime_params,
+        geometry,
+        core_profiles,
+    )
+    neoclassical_transport = self._apply_clipping(
+        runtime_params,
+        neoclassical_transport,
+    )
+    return neoclassical_transport
+
+  def _apply_clipping(
+      self,
+      runtime_params: runtime_params_slice.RuntimeParams,
+      neoclassical_transport: NeoclassicalTransport,
+  ) -> NeoclassicalTransport:
+    chi_neo_i = jnp.clip(
+        neoclassical_transport.chi_neo_i,
+        runtime_params.neoclassical.transport.chi_min,
+        runtime_params.neoclassical.transport.chi_max,
+    )
+    chi_neo_e = jnp.clip(
+        neoclassical_transport.chi_neo_e,
+        runtime_params.neoclassical.transport.chi_min,
+        runtime_params.neoclassical.transport.chi_max,
+    )
+    D_neo_e = jnp.clip(
+        neoclassical_transport.D_neo_e,
+        runtime_params.neoclassical.transport.D_e_min,
+        runtime_params.neoclassical.transport.D_e_max,
+    )
+    V_neo_e = jnp.clip(
+        neoclassical_transport.V_neo_e,
+        runtime_params.neoclassical.transport.V_e_min,
+        runtime_params.neoclassical.transport.V_e_max,
+    )
+    V_neo_ware_e = jnp.clip(
+        neoclassical_transport.V_neo_ware_e,
+        runtime_params.neoclassical.transport.V_e_min,
+        runtime_params.neoclassical.transport.V_e_max,
+    )
+    return dataclasses.replace(
+        neoclassical_transport,
+        chi_neo_i=chi_neo_i,
+        chi_neo_e=chi_neo_e,
+        D_neo_e=D_neo_e,
+        V_neo_e=V_neo_e,
+        V_neo_ware_e=V_neo_ware_e,
+    )
+
+  @abc.abstractmethod
+  def _call_implementation(
+      self,
+      runtime_params: runtime_params_slice.RuntimeParams,
+      geometry: geometry_lib.Geometry,
+      core_profiles: state.CoreProfiles,
+  ) -> NeoclassicalTransport:
+    pass
 
 
 class NeoclassicalTransportModelConfig(torax_pydantic.BaseModelFrozen, abc.ABC):
-  """Base class for neoclassical transport model configs."""
+  """Base class for neoclassical transport model configs.
+
+  Attributes:
+   chi_min: Lower bound on heat conductivity.
+   chi_max: Upper bound on heat conductivity.
+   D_e_min: minimum electron density diffusivity.
+   D_e_max: maximum electron density diffusivity.
+   V_e_min: minimum electron density convection. Note, clipping of V_neo is
+     applied to Ware and and convection terms separately.
+   V_e_max: minimum electron density convection. Note, clipping of V_neo is
+     applied to Ware and and convection terms separately.
+  """
+
+  chi_min: torax_pydantic.MeterSquaredPerSecond = 0.0
+  chi_max: torax_pydantic.MeterSquaredPerSecond = 100
+  D_e_min: torax_pydantic.MeterSquaredPerSecond = 0.0
+  D_e_max: torax_pydantic.MeterSquaredPerSecond = 100.0
+  V_e_min: torax_pydantic.MeterPerSecond = -50.0
+  V_e_max: torax_pydantic.MeterPerSecond = 50.0
+
+  @pydantic.model_validator(mode='after')
+  def _check_fields(self) -> typing_extensions.Self:
+    if not self.chi_max > self.chi_min:
```

Comment:
nit: Can you make it so that these all go the same way? It feels a bit weird to have one differently?